vocab/credentials/v2/issuer-dependent.html

```diff
@@ -12,18 +12,6 @@
     </script>
     <script class="remove">
```

Comment:
This whole file should be deleted, right? 

"issuer-dependent" is no longer a thing this WG defines.

Reply:
I was not sure about that. But yes, that is what I expected to do.

Removed in https://github.com/w3c/vc-data-model/pull/1536/commits/adaffa8248d0f560e576954449944bf7f85b12a3.

I have also removed it from the .htaccess file above.